.github/workflows/ci.yml

```diff
@@ -168,85 +168,43 @@ jobs:
     name: Functional tests
     runs-on: ubuntu-24.04
     needs: php-lint
-    env:
-      DB_DATABASE: typo3
-      DB_USER: root
-      DB_PASSWORD: root
-      DB_HOST: localhost
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-      - name: Install PHP
-        uses: shivammathur/setup-php@v2
-        with:
-          php-version: "${{ matrix.php-version }}"
-          ini-file: development
-          tools: composer:v2
-          extensions: mysqli
-          coverage: none
-      - name: Show Composer version
```

Comment:
Please let's keep these two steps (i.e., switch them to `runTests.sh`) to help debugging Composer problems.